lambdas/validate-oauth-callback/src/main/java/uk/gov/di/ipv/core/validateoauthcallback/ValidateOAuthCallbackHandler.java

```diff
@@ -130,6 +141,39 @@ public APIGatewayProxyResponseEvent handleRequest(
         }
     }
 
+    @Tracing
+    private APIGatewayProxyResponseEvent sendOauthErrorJourneyResponse(
+            IpvSessionItem ipvSessionItem, CredentialIssuerRequestDto request) throws SqsException {
+        AuditExtensionErrorParams extensions =
+                new AuditExtensionErrorParams.Builder()
+                        .setErrorCode(request.getError())
+                        .setErrorDescription(request.getErrorDescription())
+                        .build();
+        sendAuditEvent(ipvSessionItem, extensions);
+
+        if (!ALLOWED_OAUTH_ERROR_CODES.contains(request.getError())) {
+            LOGGER.warn("Unknown Oauth error code received");
+        }
+
+        ipvSessionItem.addVisitedCredentialIssuerDetails(
+                new VisitedCredentialIssuerDetailsDto(
+                        request.getCredentialIssuerId(), false, request.getError()));
+        ipvSessionService.updateIpvSession(ipvSessionItem);
+
+        if (OAuth2Error.ACCESS_DENIED_CODE.equals(request.getError())) {
+            LOGGER.info("OAuth access_denied");
```

Comment:
Could we change this to use the `LogHelper.logOauthError` method?

Reply:
So logAuthError does an `error` log but I think this one is deliberately just `info` because AccessDenied is an expected error?

Reply:
Ah okay yeah I see what you mean 👍